README.md

```diff
@@ -10,13 +10,14 @@ to make errors nice for humans.
 - [Crates.io][2]
 
 ## Why?
-When you're building a CLI, polish is super important. Even though Rust is
-pretty great at safety, it's not unheard of to access the wrong index in a
-vector or have an assert fail somewhere.
+When you're building an application, polish is super important. Even though
+Rust is pretty great at safety, it's not unheard of to access the wrong index
+in a vector or have an assert fail somewhere.
 
 When an error eventually occurs, you probably will want to know about it. So
-instead of just providing an error message on the command line, we can create a
-call to action for people to submit a report.
+instead of just providing an error message on the command line or the GUI
```

Comment:
I would re-phrase to

"[...] instead of crashing without notice or printing a confusing error message [...]"

Rust does by default print an error message but it's not super helpful. I've been meaning to change the wording here for a while anyways :wink: